product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
+---
+
+### Security Governance
+
+- Cloud Security Compliance officer is responsible for maintaining compliance targets for their entire organization.
+- EDB Cloud infrastructure periodically undergoes SOC2 audit trials to ensure that EDB Cloud is always compliant with security requirements.
+- All EDB staff undergo background checks before they are hired.
+- All EDB staff are required to take mandatory security training.
+- The EDB Software Development Life Cycle integrates continuous security scans of added/updated code.
+
+
+### EDB Cloud Security Features
+EDB Cloud is a Database as a Service (DBaaS) which runs in your own Cloud account, isolating your data from other users & giving you control over our access to it. The key security features of EDB Cloud are as follows:
+- **Segregation of IT environments:** Customers database clusters are installed and managed in their own cloud environment. Complete segregation of customer's data is ensured.
+- **Access Control:** EDB Cloud supports _Single Sign on_ and _Role Based Access Control_ policies. Customers may define their own sets of _Roles_ and  _Role Based Access Control_ policies to manage their individual cloud environments.
+ 
+- **Data Encryption:** EDB Cloud encrypts customer data in motion and at rest. All network traffic is encrypted using Transport Layer Security (TLS). Data stored in database clusters is encrypted appropriately. Users can bring their own encryption keys for an additional level of control.
+- **Logging and Auditing:** EDB Cloud supports Logging and Auditing
+- **Backups:** EDB Cloud data is continuously replicated from the production to the hot standby data center.
+- **Disaster Recovery and Business Continuity:** All components of EDB Cloud infrastructure are highly available and redundant.
+- **Incident Response:** EDB employs tools and processes which monitor the platform and has a dedicated security team as well as incident response processes.
```

Comment:
How about just three bullets here: Network isolation, Encryption, and Auditing?

BC/DR and IR are both pretty deep internal things, and backups are not really security.

Reply:
DC/DR and IR are marketing bullet points which competitors used in their docs and customers want to hear. (I lifted the top bullets from a competitor's doc which was much longer and long winded) So, If we remove DC/DR and IR  from this section we should certainly mention them somewhere else.  Not sure that High Availability section is fit for it.  Where do we put them then?

Reply:
I'd rather not put them anywhere. Dabblers can see that we have a SOC2, and serious customers can _get_ the SOC2.

Reply:
I found the doc which was the source of my inspiration for writing "Security and Compliance" section. Here it is: https://www.quickbase.com/security-and-compliance

Reply:
Yeah... That reads like a SOC2 system description to me, honestly - it's clearly organized along standard controls critera. I'm biased towards something like [MongoDB's][1].

[1]: https://www.mongodb.com/cloud/atlas/security

Reply:
I have heard rumors that MongoDB owns it success to its good marketing department,  in much more degree than  to its system design.

Reply:
...so when I have to lift content from a competitor,  I would rather emulate documentation that comes from a company with good marketing department than from a company with a good product. Obviously, it is even more  preferable to use some internal docs when they are available. Which bring me to my question: do we have some docs that can be useful to write this section in a better way?

Reply:
Can you see the System Description? It has descriptions of this sort of thing if you want something to crib from.

https://docs.google.com/document/d/1BRk1Y2RQpkcuc0Zn0AKbiVErbGFEPUZo/edit#